Automattic/wp-calypso
# Threat Alerts: Use data layer to dispatch requests to fix and ignore threats

Threat Alerts: Use data layer to dispatch requests to fix and ignore threats

Clean rebase of #34635

**Changes proposed in this Pull Request**

Instead of making one-off requests to the wpcom api from the components, we'll use redux actions and the data layer to make requests to fix and ignore threats.

**Testing instructions**

From the activity log page for a site with active threats showing, dispatch a request like so:

```
disptch( { 
	type: 'JETPACK_SITE_ALERT_THREAT_FIX',
	siteId: 123456789,
	threatId: 123456,
} );
```

You should see the threat disappear, and redux state for Rewind updated to reflect the change.
